Gemfile

```diff
@@ -3,17 +3,17 @@ gem 'rails', ' ~> 5.2.0'
 
 gem 'rack-cors', '~> 1.0.5'
 gem 'grape', '~> 1.3.2'
-
 gem 'jbuilder', '~> 2.7'
+# Virtus is no longer supported. Consider replacing with ActiveModel::Attributes
+gem 'virtus', '~> 1.0'
```

Comment:
I would not normally add an unsupported gem, but Virtus was previously required under the hood by the now-removed `elasticsearch-model` gem. Now that that gem is removed, we need to explicitly require it until we can replace it.